### ISSUE TITLE
RC Test: Ratio and Proportion 1.0.0-rc.1

### ISSUE THREAD

**Issue by zepumph** [phetsims/QA#601]
@terracoda, @emily-phet, @BLFiedler, @Matthew-Moore240, @jessegreenberg, @jbphet, @Ashton-Morris, ratio-and-proportion/1.0.0-rc.1 is ready for RC testing. 

This is the first publication for this brand new simulation. Ratio and Proportion supports the following accessibility features:
- Alternative input and keyboard navigation
- Sound
- Interactive Description
- Mobile Accessibility
- Pan and Zoom

Document issues in https://github.com/phetsims/ratio-and-proportion/issues and link to this
issue. 

Assigning to @kathy-phet and @KatieWoe for prioritization.

<details>
<summary><b>General RC Test</b></summary>

<h3>What to Test</h3>

- Click every single button.
- Test all possible forms of input.
  - Test all mouse/trackpad inputs.
  - Test all touchscreen inputs.
- If there is sound, make sure it works.
- Make sure you can't lose anything.
- Play with the sim normally.
- Try to break the sim.
- Test all query parameters on all platforms. (See [QA Book](https://github.com/phetsims/QA/blob/master/documentation/qa-book.md)
for a list of query parameters.)
- Download HTML on Chrome and iOS.
- Make sure the iFrame version of the simulation is working as intended on all platforms.
- Make sure the XHTML version of the simulation is working as intended on all platforms.
- Complete the test matrix.
- Don't forget to make sure the sim works with Legends of Learning.
- Test the Game Up harness on at least one platform.
- Check [this](https://docs.google.com/spreadsheets/d/1umIAmhn89WN1nzcHKhYJcv-n3Oj6ps1wITc-CjWYytE/edit#gid=0) LoL
spreadsheet and notify AR or AM if it not there.
- If this is rc.1 please do a memory test. (see https://github.com/phetsims/ratio-and-proportion/issues/269)
- When making an issue, check to see if it was in a previously published version
- Try to include version numbers for browsers
- If there is a console available, check for errors and include them in the Problem Description.
- As an RC begins and ends, check the sim repo. If there is a maintenance issue, check it and notify developers if
there is a problem.
- UPDATE: Please pay specific attention to hover highlighting behavior, both for mouse and (if applicable) touch.

<h3>Issues to Verify</h3>
Although this is the first RC, there are a couple of issues to be verified from previous testing.

- [x] [Simulation thinks there is a locked ratio in Discover section of R&P](https://github.com/phetsims/ratio-and-proportion/issues/299)
- [x] ["Safari Busy" when VoiceOver Enabled](https://github.com/phetsims/ratio-and-proportion/issues/267)

These issues should have the "status:ready-for-qa" label. Check these issues off and close them if they are fixed.
Otherwise, post a comment in the issue saying that it wasn't fixed and link back to this issue. If the label is
"status:ready-for-review" or "status:fixed-pending-deploy" then assign back to the developer when done, even if fixed.

<h3>Link(s)</h3>

- **[Simulation](https://phet-dev.colorado.edu/html/ratio-and-proportion/1.0.0-rc.1/phet/ratio-and-proportion_all_phet.html)**
- **[iFrame](https://phet-dev.colorado.edu/html/ratio-and-proportion/1.0.0-rc.1/phet/ratio-and-proportion_all_iframe_phet.html)**
- **[XHTML](https://phet-dev.colorado.edu/html/ratio-and-proportion/1.0.0-rc.1/phet/xhtml/ratio-and-proportion_all.xhtml)**
- **[Test Matrix](https://docs.google.com/spreadsheets/d/1UhMTOsOo4byrybtKiaRQWTXi1Zkdty_X-lHfEp8j3Pk/edit#gid=1153223089)**
- **[Legends of Learning Harness](https://developers.legendsoflearning.com/public-harness/index.html?startPayload=%7B%22languageCode%22%3A%22en%22%7D)**

<hr>

</details>

<details>
<summary><b>Accessibility (a11y) RC Test</b></summary>

<h3>What to Test</h3>

- Specific instructions can be found below.
- Make sure the a11y feature doesn't negatively affect the sim in any way.
- Test all possible forms of input.
  - Test all mouse/trackpad inputs.
  - Test all touchscreen inputs.
  - Test all keyboard navigation inputs.
  - Test all forms of input with a screen reader.
  - Test blended input using mouse/touch and keyboard.

<h3>Link(s)</h3>

- **[a11y View](https://phet-dev.colorado.edu/html/ratio-and-proportion/1.0.0-rc.1/phet/ratio-and-proportion_a11y_view.html)**
- **[Simulation](https://phet-dev.colorado.edu/html/ratio-and-proportion/1.0.0-rc.1/phet/ratio-and-proportion_all_phet.html)**
- **[Test Matrix](https://docs.google.com/spreadsheets/d/1UhMTOsOo4byrybtKiaRQWTXi1Zkdty_X-lHfEp8j3Pk/edit#gid=1153223089)**

<h3>Screen Readers</h3>

This sim supports screen readers. If you are unfamiliar with screen readers, please ask Katie to introduce you to
screen readers. If you simply need a refresher on screen readers, please consult the
[QA Book](https://github.com/phetsims/QA/blob/master/documentation/qa-book.md), which should have
all of the information you need as well as a link to a screen reader tutorial made by Jesse. Otherwise, look over the
a11y view before opening the simulation. Once you've done that, open the simulation and make sure alerts and
descriptions work as intended.

<h3>Platforms and Screen Readers to Be Tested</h3>

- [ ] Windows 10 + Latest Chrome + Latest JAWS
- [x] Windows 10 + Latest Firefox + Latest NVDA
- [x] macOS + Safari + VoiceOver
- [x] iOS + Safari + VoiceOver (only if specified in testing issue)

<h3>Critical Screen Reader Information</h3>

We are tracking known screen reader bugs in
[this Google Document](https://drive.google.com/open?id=12kTs-g-iKEIH1dyG7Q41_W_oNL4gUKbkW-IQgZjMUBw). If you find a
screen reader bug, please check it against this list.

<h3>Keyboard Navigation</h3>

This sim supports keyboard navigation. Please make sure it works as intended on all platforms by itself and with a
screen reader.

<h3>Magnification</h3>

This sim supports magnification with pinch and drag gestures on touch screens, keyboard shortcuts, and mouse/wheel controls. Please test magnfication and make sure
it is working as intended and well with the use cases of the simulation. Due to the way screen readers handle user input, magnification is NOT expected to
work while using a screen reader so there is no need to test this case.

<h3>Final Requests</h3>

- [x] If this sim is in [this list](https://github.com/phetsims/perennial/blob/master/data/accessibility), make an
issue in [website](https://github.com/phetsims/website) to ask if PhET research page links need updating. Please assign
to @terracoda and @emily-phet.

<hr>

</details>

<details>
<summary><b>FAQs for QA Members</b></summary>

<br>
  
  

  <details>
  <summary><i>There are multiple tests in this issue... Which test should I do first?</i></summary> 
  
  Test in order! Test the first thing first, the second thing second, and so on.

  </details>

  <br>

  

  <details>
  <summary><i>How should I format my issue?</i></summary>
  
  Here's a template for making issues:

      <b>Test Device</b>

      blah

      <b>Operating System</b>

      blah

      <b>Browser</b>

      blah

      <b>Problem Description</b>

      blah

      <b>Steps to Reproduce</b>

      blah

      <b>Visuals</b>

      blah

      <details>
      <summary><b>Troubleshooting Information</b></summary>

      blah

      </details>

  </details>

  <br>

  

  <details>
  <summary><i>Who should I assign?</i></summary>
  
  We typically assign the developer who opened the issue in the QA repository.

  </details>
  
  <br>

  

  <details>
  <summary><i>My question isn't in here... What should I do?</i></summary>
  
  You should:

  1. Consult the [QA Book](https://github.com/phetsims/QA/blob/master/documentation/qa-book.md).
  2. Google it.
  3. Ask Katie.
  4. Ask a developer.
  5. Google it again.
  6. Cry.

  </details>

<br>

<hr>

</details>

**Comment by KatieWoe** [comment]
Memory test results:
<img width="428" alt="memtest" src="https://user-images.githubusercontent.com/41024075/106791049-aa259900-6611-11eb-8910-ef2262ed4162.png">

**Comment by KatieWoe** [comment]
QA is done. Jaws was more partially tested due to https://github.com/phetsims/ratio-and-proportion/issues/348, but I believe @loganbraywork spent some time on it as well.

**Comment by Ashton-Morris** [comment]
There is no UI sound for checking or unchecking the  "Enhanced Sound" option in the lower right speech bubble menu. 
Since Enhanced Sound would be used for a population where sound was important I think we should add a UI sound. 

I suggest using the standard Checkbox and UncheckBox sounds which is the sound used for the "Lock Ratio" option.

**Comment by emily-phet** [comment]
@Ashton-Morris nice catch!

After this initial publication, there will be a following publication that will move the "enhanced sound" option into the new Preferences menu Jesse is building. Let's not add sound to the current "enhanced sound" selection, but consider sound for the Preferences and Quick Menus. We might have a working version of this to demo for this Tuesday's sound meeting, and I can show you the latest then.

**Comment by jessegreenberg** [comment]
@zepumph @KatieWoe ratio-and-porportion 1.0 scenery SHA received a patch for phetsims/phet-android-app#155.

If possible, please verify this sim works well in the PhET Android app as part of the next RC test.

**Comment by zepumph** [comment]
All is done here, I'll note https://github.com/phetsims/QA/issues/601#issuecomment-777986115 in the next RC issue, and make a new RC, see https://github.com/phetsims/ratio-and-proportion/issues/337
